apps/test/unit/templates/sectionProgress/progressTables/ProgressTableContainerTest.jsx

```diff
@@ -37,8 +40,23 @@ const DEFAULT_PROPS = {
 };
 
 const setUp = (overrideProps = {}) => {
+  const store = createStore(
+    combineReducers({
+      sectionProgress
+    }),
+    {
+      sectionProgress: {
+        showSectionProgressDetails: false
+      }
+    }
+  );
+
   const props = {...DEFAULT_PROPS, ...overrideProps};
-  return mount(<ProgressTableContainer {...props} />);
+  return mount(
+    <Provider store={store}>
+      <ProgressTableContainer {...props} />
```

Comment:
this combination of unconnected component with provider feels weird to me. since you're using the unconnected component it feels like `showSectionProgressDetails` should be passed in as a prop. since this test isn't even testing anything about that prop though, can we handle this with default props on the components that need it instead?

Reply:
Yeah, this was an unfortunate change. The issue is that these components are set up in a way where we can't render them with a shallow mount (shallow mount would mean children aren't rendered). Because of that, the whole tree has to be rendered and down the tree a "connected" component is imported. I don't think this is something we can resolve with props. The issue is not that a prop is missing, but that down the tree we're attempting to connect to a store.

Reply:
I think we'd either have to figure out how we could adjust this component so that it can be shallow mounted, thread the prop through instead of connecting to the store in the child or figure out how to always import the unconnected component instead of default in the test environment?

Reply:
It looks like we're setting state of child components in `ProgressTableContainer`, so maybe I can do some refactoring and then use shallow mounting, I'm going to try that.

Reply:
Update: I made a couple changes and was able to resolve this.

Reply:
nicely done! simple change to the component and presumably using `shallow` instead of `mount` will make those tests run faster~